lib/gemstash/migrations/01_gem_dependencies.rb

```diff
@@ -19,9 +19,9 @@
       TrueClass :prerelease, :null => false
       DateTime :created_at, :null => false
       DateTime :updated_at, :null => false
-      index [:rubygem_id, :number, :platform], :unique => true
+      index %i[rubygem_id number platform], :unique => true
```

Comment:
Introducing this syntax places our compatibility clearly in Ruby 2.

Another PR should update the gemspec with a `required_ruby_version` marker.

Reply:
did not think about that is there a desire to avoid requiring 2+ still?